.idea/.name

```diff
@@ -1 +0,0 @@
-SampleProtractor
```

Comment:
I removed .idea folder from git repository because people don't have to use WebStorm / PHPStorm.